spec/models/plan_statistics_spec.rb

```diff
@@ -269,7 +269,10 @@
     end
     describe "#planned_completion_date_pretty" do
       it "contains the correct month and year" do
-        expect(subject.planned_completion_date_pretty).to eq "August 2014"
+        pretty_month = completion_quarter.to_date(true).strftime('%B')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.